catroid/src/org/catrobat/catroid/pocketmusic/note/trackgrid/GridRow.java

```diff
@@ -52,7 +51,7 @@ public int hashCode() {
 		int result = 21;
 		result = 31 * result + noteName.hashCode();
```

Comment:
As far as I remember the book Effective Java, the number which is used as factor for "result" is arbitrary but has to be prime. 
I think anyone who hasn't read this book may could be confused where the numbers 21 and 31 come from so we may should give them names?